uploads/upload.ts

```diff
@@ -0,0 +1,92 @@
+const upload = require("express").Router();
+const express = require("express");
+const dotenv = require("dotenv");
+import { Project } from "../models/Model";
+
+dotenv.config();
+
```

Comment:
We also have the dotenv config in the index file, I don't believe it is needed here. But I could be wrong on that as well